Userland/Services/SpiceAgent/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ serenity_component(
 
```

Comment:
SpiceAgent: Let's start rewriting the messaging system :^)

This commit could use some description on the commit itself explaining the motivation or goals of this new messaging design.